README.md

```diff
@@ -99,9 +99,44 @@ would in a Django shell.
 
 ### Running tests
 
-Unlike our other web applications this project is not using tox. To run tests locally, run this command instead:
+### NOTE: These commands can be run with ```docker-compose exec``` to execute them in an already-running container, or with ```docker-compose run --rm``` to execute them in a new container.
```

Comment:
Use four "#"s to make this note smaller than the section header: `#### NOTE: These commands...`